app/components/HeaderSlider/index.tsx

```
@@ -0,0 +1,51 @@
+"use client";
+
+import { useEffect, useState } from 'react';
+import styles from './HeaderSlider.module.css';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)